libs/ui/lib/message/Message.tsx

```diff
@@ -70,6 +72,7 @@ export const Message = ({
   className,
   variant = 'success',
   cta,
+  alternateIcon,
```

Comment:
```suggestion
  icon: alternateIcon,
```

Small nit here, but from the perspective of the consumer of this component they have the option to provide an icon or a default will be used... so I think having this just be `icon` would be sufficient. Non-blocking, fine if you just want to move forward as is.